docs/api/qiskit/0.46/qiskit.algorithms.optimizers.NFT.mdx

```diff
@@ -28,10 +28,14 @@ python_api_name: qiskit.algorithms.optimizers.NFT
 
   **Notes**
 
-  In this optimization method, the optimization function have to satisfy three conditions written in [\[1\]](#id2).
+  In this optimization method, the optimization function have to satisfy three conditions written in   <span id="id1" />[\[1\]](#id2).
 
   **References**
 
+  ## Footnotes
```

Comment:
We probably shouldn't be adding the h2 here: there is already the references description right above, and h2 messes up the information hierarchy.